### PR TITLE
Minor improvements for the permalink Copy to clipboard button

### PR DESCRIPTION
**Pull request by afercia**
Waiting for #5494, I'd like to propose a few minor improvements for the permalink "Copy" button.

This PR tries to re-use existing component and patterns. Specifically:
- uses an `IconButton` component instead of a `Button`
- uses `classnames()` to conditionally apply the `is-copied` CSS class 
- shortens the label to 'Copy the permalink'
- makes the button style more similar to default icon buttons (see border-radius)
- improves the button alignment

Screenshot before:

before

After:

after

after2

### CODE REVIEW

**Review by youknowriad — COMMENTED** [review]
Nice improvement, maybe we should hide the borders completely when the button has the `is-copied` class. It feels a bit strange to be able to hover it even if disabled.

**Comment by afercia** [comment]
@youknowriad thanks. In the latest commit:
- the simpler way to reset the styles when the permalink is copied is to "disable" the button: the CSS already provided some styles for disabled buttons on `hover`, I've applied the same to `focus`
- to avoid a focus loss, I'm using `aria-disabled` instead of `disabled`
- updated the label/tooltip when the permalink is copied

Question: should the button "noop" when the permalink is already copied? Not sure...
After all, I'm not sure "disabling" the button is a good idea in the first place. For example, if I copy the permalink, then I edit the permalink and I want to copy it again before saving the draft or updating the post, I can't do that: the button still looks "disabled"

TIL: updating the tooltip to empty string or `null` produces a focus loss, I guess because the whole parent + children get re-rendered.

**Review by youknowriad — COMMENTED** [review]
Maybe we shouldn't really disable the button but make it look disabled. I'm asking because right now when I copy, I don't see the tooltip "Permalink copied" I have to move away from the button and hover back.

**Comment by afercia** [comment]
> Maybe we shouldn't really disable the button

@youknowriad it's not disabled :) 

> when I copy, I don't see the tooltip "Permalink copied" I have to move away from the button and hover back.

that's just how the tooltip works on any button: it disappears when clicking, also on master.

**Comment by youknowriad** [comment]
I guess we should leave the tooltip fix for another PR in that case :)

**Review by youknowriad — APPROVED** [review]
LGTM 👍

**Comment by afercia** [comment]
Thanks!

File: components/clipboard-button/index.js [hunk 1]
```diff
@@ -82,9 +82,9 @@ class ClipboardButton extends Component {
 
 		return (
 			<span ref={ this.bindContainer }>
-				<Button { ...buttonProps } className={ classes }>
+				<IconButton { ...buttonProps } icon="admin-links" className={ classes }>
```
Comment (aduth) [inline components/clipboard-button/index.js]: Should this have been applied to ClipboardButton broadly, or is it specific to PostPermalink ?

It's affecting other usage, such as the ClipboardButtons shown in the editor crash prompt:

image

Comment (youknowriad) [reply components/clipboard-button/index.js]: Oh I missed that sorry :(

**Comment by afercia** [comment]
Will have a look.

**Pull request merged**